content/en/docs/contribute/style/style-guide.md

```diff
@@ -411,7 +411,7 @@ Website navigation links can also be marked up as list items; after all they are
 
 ### Tables
 
-The semantic purpose of a data table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line. Proper markup must be added to help the screen reader make the correct associations that a sighted user would.
+The semantic purpose of a data table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line. Imagine reading an entire table cell by cell and then piecing together what the table is meant for. Hence, it is important to add a proper markup such as a table caption to help the screen reader make the correct associations that a sighted user would. 
```

Comment:
nit picky: Keep it simple. Do you have any advice on how to write a well written table caption?
Possible edits/ideas:
A table caption is used to create a descriptive title for a data table. Assistive technologies (AT) use the HTML table caption element to identify the table contents to the user within the page structure.